src/main/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/GermlineCNVSegmentVariantComposer.java

```diff
@@ -155,8 +155,14 @@ VariantContext composeVariantContext(final IntegerCopyNumberSegment segment) {
         genotypeBuilder.attribute(QSE, FastMath.round(segment.getQualityEnd()));
         final Genotype genotype = genotypeBuilder.make();
 
+        final List<Allele> vcAlleles = new ArrayList<>(Collections.singletonList(REF_ALLELE));
```

Comment:
Could write this as `final List<Allele> vcAlleles = Lists.newArrayList(REF_ALLELE);`

Reply:
`Lists` is in which package?  avro?